Exiled.Events/EventArgs/Scp106/PlayerTryLeaveStalkEventArgs.cs

```diff
@@ -0,0 +1,87 @@
+// -----------------------------------------------------------------------
+// <copyright file="PlayerTryLeaveStalkEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Scp106
+{
+    using API.Features;
+    using Interfaces;
+    using Mirror;
+    using PlayerRoles.PlayableScps.Scp106;
+    using PlayerRoles.PlayableScps.Subroutines;
+    using UnityEngine;
+
+    /// <summary>
+    ///     Contains all information before SCP-106 teleports using a portal.
+    /// </summary>
+    public class PlayerTryLeaveStalkEventArgs : IPlayerEvent, IDeniableEvent
```

Comment:
Rename to FinishingStalkingEventArgs

Reply:
Same as above

Reply:
`RequestingEndStalkEventArgs`? As per previous suggestion

Reply:
Done